examples/simple.ts

```diff
@@ -0,0 +1,38 @@
+import { OpenFeature } from '@openfeature/server-sdk';
+import  { HyphenProvider, type HyphenProviderOptions, type HyphenEvaluationContext } from 'openfeature-provider-javascript-server';
+
+const publicKey = "your-public-key";
+
+const options: HyphenProviderOptions = {
+  application: 'application-id',
+  environment: 'production',
+}
+
+const context: HyphenEvaluationContext = {
+  ...options,
+  targetingKey: 'target-key-1',
```

Comment:
we are going to recommend that the targeting key is the same as the user.id (if there is a user) so these should match. 
```suggestion
  targetingKey: 'user-123',
```